script/install_07_polyaxon.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+info() {
+    echo "$(tput bold)====> $1$(tput sgr0)"
+}
+
+# This file is based on https://docs.polyaxon.com/setup/archlinux-kubeadm.
+
+info "Add polyaxon repository to helm."
+helm repo add polyaxon https://charts.polyaxon.com
+helm repo update polyaxon
+
+info "Create polyaxon namespace."
+kubectl create namespace polyaxon
+
+# TODO Use ingress
+# TODO SSL
+# TODO Persistence
+# TODO Enable nvidia support. See https://docs.polyaxon.com/setup/archlinux-kubeadm/#add-support-for-nvidia-framework-to-docker.
+# TODO Add script for adding a worker node.
```

Comment:
The last 3 are the most important ones.

Reply:
Certainly. Persistence is vital to not lose our data ofc. Nvidia cause AI.

Reply:
Also yeah, making this cluster easily extensible to the other workstations is ideal.